BE/src/main/resources/schema.sql

```diff
@@ -0,0 +1,49 @@
+DROP TABLE IF EXISTS column;
+DROP TABLE IF EXISTS `user`;
+DROP TABLE IF EXISTS card;
+DROP TABLE IF EXISTS log;
+
+CREATE TABLE column
+(
+    id       INTEGER AUTO_INCREMENT NOT NULL,
+    name     VARCHAR(255) NOT NULL,
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE `user`
+(
+    id       VARCHAR(255) NOT NULL,
+    name     VARCHAR(255) NOT NULL,
+    password VARCHAR(255) NOT NULL,
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE card
+(
+     id BIGINT AUTO_INCREMENT NOT NULL,
+     user_id VARCHAR(255) NOT NULL,
+     column_id INTEGER NOT NULL,
+     subject VARCHAR(255) NOT NULL,
+     contents VARCHAR(255) NOT NULL,
+     create_time TIMESTAMP NOT NULL,
+     update_time TIMESTAMP,
+     deleted boolean DEFAULT FALSE,
+     PRIMARY KEY (id),
+     FOREIGN KEY (user_id) REFERENCES `user` (id),
+     FOREIGN KEY (column_id) REFERENCES column (id)
```

Comment:
공백과 들여쓰기를 적절히 사용해서 코드를 읽기 쉽게 하면 좋을 것 같아요~ 예시는 참고만 해주세요!
```suggestion
id          BIGINT AUTO_INCREMENT NOT NULL,
user_id     VARCHAR(255) NOT NULL,
column_id   INTEGER      NOT NULL,
subject     VARCHAR(255) NOT NULL,
contents    VARCHAR(255) NOT NULL,
create_time TIMESTAMP    NOT NULL,
update_time TIMESTAMP,
deleted     boolean DEFAULT FALSE,
PRIMARY KEY (id),
FOREIGN KEY (user_id) REFERENCES `user` (id),
FOREIGN KEY (column_id) REFERENCES column (id)
```

Reply:
오 넵! 가독성이 훨씬 좋아보이네요. 수정하겠습니다. 감사합니다. !